autotest/osr/osr_basic.py

```diff
@@ -1663,6 +1663,17 @@ def test_osr_promote_to_3D():
     assert sr.GetAuthorityCode(None) == '4326'
 
 
+def test_osr_strip_vertical():
+
+    sr = osr.SpatialReference()
+    sr.ImportFromEPSG(9528)  # NZGD2000 + NZVD2016 height
```

Comment:
EPSG:9528 is probably too recent for the oldest PROJ release we test. I'd suggest using instead
sr.ImportFromEPSG(3901) # "KKJ / Finland Uniform Coordinate System + N60 height"
already available in PROJ 6
The horizontal code is 2393